Exoft.Gamification.Api.Data.Core/Interfaces/IAchievementRepository.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using Exoft.Gamification.Api.Data.Core.Entities;
 using Exoft.Gamification.Api.Data.Core.Helpers;
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Exoft.Gamification.Api.Data.Core.Interfaces
 {
     public interface IAchievementRepository : IRepository<Achievement>
     {
-        Task<ICollection<Achievement>> GetPagedAchievementAsync(PageInfo pageInfo);
+        Task<Achievement> IsUserHaveAchievementAsync(Guid userId, Guid achievementId);
```

Comment:
We don't need this method? (IsUserHaveAchievementAsync)

Reply:
also DoesUserHave - is correct... not Is

Reply:
Well.... I've looked on further implementation and understood that it is just bad name... so please give proper name